### PR TITLE
Implement Json skeleton files

### PR DESCRIPTION
**Pull request by luffingluffy**

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 1]
```diff
@@ -94,16 +131,26 @@ public Person toModelType() throws IllegalValueException {
...
+        final Object modelShnPeriod = shnPeriod;
+        final Object modelCaseNumber = caseNumber;
+        final Object modelNextOfKinName = nextOfKinName;
+        final Object modelNextOfKinPhone = nextOfKinPhone;
+        final Object modelNextOfKinAddress = nextOfKinAddress;
 
         final Set<Tag> modelTags = new HashSet<>(personTags);
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelTags);
+        return new Person(modelName, modelPhone, modelEmail, modelHomeAddress, modelQuarantineAddress,
+                modelWorkAddress, modelShnPeriod, modelCaseNumber, modelNextOfKinName, modelNextOfKinPhone,
```
Comment (lrnshk) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java:152]: The order here is wrong. Person constructor takes workAddress before quarantineAddress.

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 2]
```diff
@@ -27,20 +27,50 @@
...
      */
+    @Deprecated
+    public JsonAdaptedPerson(String name, String phone, String email, String homeAddress, List<JsonAdaptedTag> tagged) {
+        this(name, phone, email, homeAddress, null, null, null, null, null, null, null, tagged);
+    }
+
+    /**
+     * Constructs a {@code JsonAdaptedPerson} with the given person details.
+     */
+    // TODO: JSON property for "address" to be renamed to "homeAddressed" when integrating
```
Comment (lrnshk) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java:51]: homeAddress*

**Review by lrnshk — CHANGES_REQUESTED** [review]
Minor issues

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 3]
```diff
@@ -94,16 +131,26 @@ public Person toModelType() throws IllegalValueException {
...
         }
-        if (!Address.isValidAddress(address)) {
+        if (!Address.isValidAddress(homeAddress)) {
             throw new IllegalValueException(Address.MESSAGE_CONSTRAINTS);
         }
-        final Address modelAddress = new Address(address);
+        final Address modelHomeAddress = new Address(homeAddress);
+
+        final Object modelQuarantineAddress = quarantineAddress;
+        final Object modelWorkAddress = workAddress;
```
Comment (wei2912) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java:143]: Order of these two lines should be swapped.

**Review by wei2912 — CHANGES_REQUESTED** [review]

**Review by wei2912 — APPROVED** [review]

**Pull request merged**